src/pages/HomePage.vue

```diff
@@ -5,7 +5,7 @@
       <OverviewSkeleton v-else  class="sticky" />
     </div>
     <div class="justify-start items-start" :style="columnStyle">
-      <threads v-if="activeRoom?.threads" :threads="activeRoom?.threads"/>
+      <threads v-if="threads" :threads="threads" @onLoad="onLoad"/>
```

Comment:
```suggestion
      <Threads v-if="threads" :threads="threads" @onLoad="onLoad"/>
```

Reply:
does it need `v-if="threads"` ? I assume the getter returns an empty array and then this Threads doesn't do anything that bad if it's got an empty array? oh.... that spinner will show the whole time I guess. meh